File: app/src/main/java/net/imatruck/dsmanager/models/TaskStatusExtra.java
package net.imatruck.dsmanager.models;

import com.google.gson.annotations.SerializedName;


@SuppressWarnings("unused")
public class TaskStatusExtra {

    @SerializedName("error_detail")
    private String errorDetail;

    @SerializedName("unzip_progress")
    private int unzipProgress;
}

File: app/src/main/java/net/imatruck/dsmanager/models/TaskStatusExtra.kt
package net.imatruck.dsmanager.models

import com.google.gson.annotations.SerializedName


class TaskStatusExtra {

    @SerializedName("error_detail")
    private val errorDetail: String? = null

    @SerializedName("unzip_progress")
    private val unzipProgress: Int = 0
}
